### PR TITLE
Add Playstation

### PR DESCRIPTION
**Pull request by Raymo111**

### CODE REVIEW

**Review by kallentu — APPROVED** [review]
Awesome, thanks!

File: README.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ Good luck and happy hunting :tada:!
 | [Microsoft (US)](https://careers.microsoft.com/students/us/en/job/1085294/Software-Engineering-Intern-Opportunities) | SWE | Multiple locations | | |
 | [Microsoft (Canada)](https://careers.microsoft.com/us/en/job/1116547/Software-Engineer-Winter-Co-op-Intern-Opportunities%E2%80%AF%E2%80%AF) | SWE | Vancouver, Canada | | |
 | [Palantir](https://jobs.lever.co/palantir/5d5ff415-8219-4e0c-9930-2d5919e90354) | SWE | London, New York, Palo Alto, Washington | | |
+| [Playstation (US & Canada)](https://boards.greenhouse.io/sonyinteractiveentertainmentplaystation/jobs/3316438) | SWE, SDET, multiple other jobs | Cali, other US and Waterloo | | |
```
Comment (kallentu) [inline README.md]: Actually, can you just put `Multiple locations` for the locations. Unless you can specify where in California, city-wise.

**Review by kallentu — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Raymo111) [reply README.md]: Kay

Comment (Raymo111) [reply README.md]: @kallentu updated

**Review by kallentu — APPROVED** [review]

**Pull request merged**